src/routes/_pages/settings/index.html

```diff
@@ -11,6 +11,9 @@ <h1>Settings</h1>
     <SettingsListRow>
       <SettingsListButton href="/settings/hotkeys" label="Hotkeys"/>
     </SettingsListRow>
+    <SettingsListRow>
+    <SettingsListButton href="" on:click=reload(event) label="Reload"/>
```

Comment:
This href should be a `#` if it's a dynamic event. I forget why, but I think this is what you're supposed to do :joy: